apis/task.py

```diff
@@ -90,14 +97,14 @@ def provision(self) -> Tuple[DAGNode, airflow.XComArg, airflow.XComArg]:
     """
     with TaskGroup(group_id="provision") as group:
       with TaskGroup(group_id="initialize"):
-        if self.task_test_config.custom_tpu_name.strip():
-          base_tpu_name = self.task_test_config.custom_tpu_name
+        if self.custom_tpu_name.strip():
```

Comment:
Why `strip` here?

Reply:
Just in case anyone accidentally writes space for a custom name.